app/orm-adapters/node-orm2.js

```diff
@@ -7,7 +7,7 @@ import snakeCase from 'lodash/snakeCase';
 import upperFirst from 'lodash/upperFirst';
 
 const typeMap = {
-  text: String,
+  string: String,
```

Comment:
@davewasmer Understood and sounds good.  The reason I changed this here though was that the add-on was failing with errors about a type not being found. Once this line was changed to `string`, I stopped getting errors (and seemed to have a functioning orm). Will check into this further ...